### DISCUSSION TITLE
Compile error

### DISCUSSION THREAD

**Discussion by brz-hub** [CodexLabsLLC/Colosseum#16] [General]
Latest Colcosseum version
UE/Unity version: 5.0.3
OS Version: win10

3 error：
error C2672: “common_utils::Utils::isDefinitelyLessThan”: 未找到匹配的重载函数 （No matching overloaded function found）
error C2027: 使用了未定义类型“physx::PxVehicleWheels” （Type not defined）
error C2027: 使用了未定义类型“physx::PxVehicleDrive” （Type not defined）

with NET Framework3.0 Installed，I successfully ran the build.cmd command on "Colosseu" folder.However, when I “Switch Unreal Engine Version to ue5.0"  and then  tried to compile it, I encountered the above problem, which was also encountered in the process  of compiling AirSim with UE5.

**Comment by xxEoD2242** [comment]
What branch are you on? We have a ue5 branch that has the corrections needed

**Reply by brz-hub** [reply]
I'm sorry, I used the wrong branch.The UE5 branch  works properly.

**Reply by xxEoD2242** [reply]
Awesome! Glad to hear it.

Let us know if you find anything that is wrong with the new implementation.

**Reply by brz-hub** [reply]
> Awesome! Glad to hear it.
> 
> Let us know if you find anything that is wrong with the new implementation.

I will. And thank you very much for your reply, and I'm sorry for my low-level mistakes again.
